src/gu/sceGuInit.c

```diff
@@ -12,39 +12,266 @@
+		ZV(Z_TEST_ENABLE),
+		ZV(STENCIL_TEST_ENABLE),
+		ZV(ANTI_ALIAS_ENABLE),
+		ZV(PATCH_CULL_ENABLE),
+		ZV(COLOR_TEST_ENABLE),
+
+		ZV(LOGIC_OP_ENABLE),
+		ZV(BONE_MATRIX_NUMBER),
+		ZV(BONE_MATRIX_DATA),
+		ZV(MORPH_WEIGHT0),
+		ZV(MORPH_WEIGHT1),
+		ZV(MORPH_WEIGHT2),
+		ZV(MORPH_WEIGHT3),
+		ZV(MORPH_WEIGHT4),
+
+		ZV(MORPH_WEIGHT5),
+		ZV(MORPH_WEIGHT6),
+		ZV(MORPH_WEIGHT7),
+		ZV(PATCH_DIVISION),
+		ZV(PATCH_FACING),
+		ZV(PATCH_PRIMITIVE),
+		ZV(WORLD_MATRIX_NUMBER),
+		ZV(WORLD_MATRIX_DATA),
+
+		ZV(VIEW_MATRIX_NUMBER),
+		ZV(VIEW_MATRIX_DATA),
+		ZV(PROJ_MATRIX_NUMBER),
+		ZV(PROJ_MATRIX_DATA),
+		ZV(TGEN_MATRIX_NUMBER),
+		ZV(TGEN_MATRIX_DATA),
+		ZV(VIEWPORT_X_SCALE),
+		ZV(VIEWPORT_Y_SCALE),
+
+		ZV(VIEWPORT_Z_SCALE),
+		ZV(VIEWPORT_X_CENTER),
+		ZV(VIEWPORT_Y_CENTER),
+		ZV(VIEWPORT_Z_CENTER),
+		ZV(TEX_SCALE_U),
+		ZV(TEX_SCALE_V),
+		ZV(TEX_OFFSET_U),
+		ZV(TEX_OFFSET_V),
+
+		ZV(OFFSET_X),
+		ZV(OFFSET_Y),
+		ZV(SHADE_MODE),
+		ZV(NORMAL_REVERSE),
+		ZV(MATERIAL_COLOR),
+		ZV(MATERIAL_EMISSIVE),
+		ZV(AMBIENT_COLOR),
+		ZV(MATERIAL_DIFFUSE),
+
+		ZV(MATERIAL_SPECULAR),
+		ZV(AMBIENT_ALPHA),
+		ZV(MATERIAL_SPECULAR_COEF),
+		ZV(AMBIENT_LIGHT_COLOR),
+		ZV(AMBIENT_LIGHT_ALPHA),
+		ZV(LIGHT_MODE),
+		ZV(LIGHT_TYPE0),
+		ZV(LIGHT_TYPE1),
+
+		ZV(LIGHT_TYPE2),
+		ZV(LIGHT_TYPE3),
+		ZV(LIGHT0_X),
+		ZV(LIGHT0_Y),
+		ZV(LIGHT0_Z),
+		ZV(LIGHT1_X),
+		ZV(LIGHT1_Y),
+		ZV(LIGHT1_Z),
+
+		ZV(LIGHT2_X),
+		ZV(LIGHT2_Y),
+		ZV(LIGHT2_Z),
+		ZV(LIGHT3_X),
+		ZV(LIGHT3_Y),
+		ZV(LIGHT3_Z),
+		ZV(LIGHT0_DIRECTION_X),
+		ZV(LIGHT0_DIRECTION_Y),
+
+		ZV(LIGHT0_DIRECTION_Z),
+		ZV(LIGHT1_DIRECTION_X),
+		ZV(LIGHT1_DIRECTION_Y),
+		ZV(LIGHT1_DIRECTION_Z),
+		ZV(LIGHT2_DIRECTION_X),
+		ZV(LIGHT2_DIRECTION_Y),
+		ZV(LIGHT2_DIRECTION_Z),
+		ZV(LIGHT3_DIRECTION_X),
+
+		ZV(LIGHT3_DIRECTION_Y),
+		ZV(LIGHT3_DIRECTION_Z),
+		ZV(LIGHT0_CONSTANT_ATTEN),
+		ZV(LIGHT0_LINEAR_ATTEN),
+		ZV(LIGHT0_QUADRATIC_ATTEN),
+		ZV(LIGHT1_CONSTANT_ATTEN),
+		ZV(LIGHT1_LINEAR_ATTEN),
+		ZV(LIGHT1_QUADRATIC_ATTEN),
 
+		ZV(LIGHT2_CONSTANT_ATTEN),
+		ZV(LIGHT2_LINEAR_ATTEN),
+		ZV(LIGHT2_QUADRATIC_ATTEN),
+		ZV(LIGHT3_CONSTANT_ATTEN),
+		ZV(LIGHT3_LINEAR_ATTEN),
+		ZV(LIGHT3_QUADRATIC_ATTEN),
+		ZV(LIGHT0_EXPONENT_ATTEN),
+		ZV(LIGHT1_EXPONENT_ATTEN),
+
+		ZV(LIGHT2_EXPONENT_ATTEN),
+		ZV(LIGHT3_EXPONENT_ATTEN),
+		ZV(LIGHT0_CUTOFF_ATTEN),
+		ZV(LIGHT1_CUTOFF_ATTEN),
+		ZV(LIGHT2_CUTOFF_ATTEN),
+		ZV(LIGHT3_CUTOFF_ATTEN),
+		ZV(LIGHT0_AMBIENT),
+		ZV(LIGHT0_DIFFUSE),
+
+		ZV(LIGHT0_SPECULAR),
+		ZV(LIGHT1_AMBIENT),
+		ZV(LIGHT1_DIFFUSE),
+		ZV(LIGHT1_SPECULAR),
+		ZV(LIGHT2_AMBIENT),
+		ZV(LIGHT2_DIFFUSE),
+		ZV(LIGHT2_SPECULAR),
+		ZV(LIGHT3_AMBIENT),
+
+		ZV(LIGHT3_DIFFUSE),
+		ZV(LIGHT3_SPECULAR),
+		ZV(CULL),
+		ZV(FRAME_BUF_PTR),
+		ZV(FRAME_BUF_WIDTH),
+		ZV(Z_BUF_PTR),
+		ZV(Z_BUF_WIDTH),
+		ZV(TEX_ADDR0),
+
+		ZV(TEX_ADDR1),
+		ZV(TEX_ADDR2),
+		ZV(TEX_ADDR3),
+		ZV(TEX_ADDR4),
+		ZV(TEX_ADDR5),
+		ZV(TEX_ADDR6),
+		ZV(TEX_ADDR7),
+		CV(TEX_BUF_WIDTH0, 512 * 512),
```

Comment:
```suggestion
		CV(TEX_BUF_WIDTH0, 512 * 512) + 4,
```
I don't understand this, but like this this one matches. I don't know if there is some bug here or something in the original.

Reply:
Done!